nilearn/plotting/_utils.py

```diff
@@ -77,19 +86,44 @@ def _check_bg_map(bg_map, hemi):
     bg_map : str | pathlib.Path | numpy.ndarray | None
     """
     if isinstance(bg_map, SurfaceImage):
-        assert bg_map.data.parts[hemi] is not None
         if len(bg_map.shape) > 1 and bg_map.shape[1] > 1:
             raise TypeError(
                 "Input data has incompatible dimensionality. "
                 f"Expected dimension is ({bg_map.shape[0]},) "
                 f"or ({bg_map.shape[0]}, 1) "
                 f"and you provided a {bg_map.shape} surface image."
             )
-        bg_map = bg_map.data.parts[hemi]
+        if hemi == "both":
+            bg_map = np.concatenate(
```

Comment:
same here

Reply:
this is again the data for the background image